File: src/SOP8/Egovframework/Base/base-model/src/main/java/egovframework/base/model/sop/category/LargeClass.java
package egovframework.base.model.sop.category;

import lombok.*;
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class LargeClass {
	public enum Fields { lclas_sn, lclas_name, site_sn }

	private int lclas_sn;

	@NonNull
	@Builder.Default
	private String lclas_name = "";

	private int site_sn;

	public static final String autoIncreaseField = "lclas_sn";
	public static final String tableName = "so_ctgry_lclas";
	public static final String[] primaryKeys = { "lclas_sn" };

}

File: src/SOP8/Base/Base.Model/Sop/Category/LargeClass.cs
using System;
using dnsDapperDBUtil.DataAccessLayer.IDAL;

namespace Base.Model.Sop.Category
{
	public class LargeClass : Table
	{
		public enum Fields { lclas_sn, lclas_name, site_sn };
		public enum WriteFields { lclas_name, site_sn };

		public int lclas_sn { get; set; }
		public string lclas_name { get; set; }
		public int site_sn { get; set; }

		public static string TableName { get { return "so_ctgry_lclas"; } }

		public override string GetTableName()
		{
			return TableName;
		}

		public override string GetPrimaryCondition()
		{
			return string.Format("{0} = {1}", Fields.lclas_sn, lclas_sn);
		}

		public override Type GetFieldType()
		{
			return typeof(Fields);
		}

		public override Type GetWriteFieldType()
		{
			return typeof(WriteFields);
		}

		public void FromCopy(LargeClass obj)
		{
			this.lclas_sn = obj.lclas_sn;
			this.lclas_name = obj.lclas_name;
			this.site_sn = obj.site_sn;
		}
	}
}
